engines/tinsel/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
 # add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps] [components]
-add_engine tinsel "Tinsel" yes "" "" "" "mt32emu"
+add_engine tinsel "Tinsel" yes "" "" "" "mt32emu tinygl"
```

Comment:
I don't think Tinsel needs tinygl

Reply:
It does, for the 3D protagonist model in DW: Noir

Reply:
It might be a good idea to split Discworld Noir support into a subengine, depending on how much additional code it requires.

Reply:
> It does, for the 3D protagonist model in DW: Noir

Indeed, I wonder how I managed to fail my grep so hard. :D